executor/executor_freebsd.cc

```diff
@@ -20,11 +20,23 @@
 #endif
 
 #include <signal.h>
+
+#include <fcntl.h>
+
+#include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/resource.h>
 #include <sys/time.h>
 #include <sys/types.h>
 
+#define KIOENABLE	_IOW('c', 2, int)	/* Enable coverage recording */
```

Comment:
This needs to be reformatted with clang-format, CI has failed on this:
https://travis-ci.org/google/syzkaller/jobs/361843688
'make format' should reformat everything, or if make does not work just extract the clang-format command line from the Makefile.